runtime/oti/j9consts.h

```diff
@@ -485,7 +485,8 @@ extern "C" {
 
 #define J9_GC_MINIMUM_OBJECT_SIZE 0x10
 #define J9_GC_INDEXABLE_DATA_FIELD_SIZE sizeof(void *)
-#define J9_GC_MINIMUM_INDEXABLE_OBJECT_SIZE (J9_GC_MINIMUM_OBJECT_SIZE + J9_GC_INDEXABLE_DATA_FIELD_SIZE)
+#define J9_GC_MINIMUM_INDEXABLE_OBJECT_SIZE_WITH_DATA_FIELD (J9_GC_MINIMUM_OBJECT_SIZE + J9_GC_INDEXABLE_DATA_FIELD_SIZE)
+#define J9_GC_MINIMUM_INDEXABLE_OBJECT_SIZE (J9_GC_MINIMUM_OBJECT_SIZE)
```

Comment:
Hm, this is risky in a long term since expects from a user to understand what dataAddr field is and when it's used
Safer is to just generalize J9_GC_MINIMUM_INDEXABLE_OBJECT_SIZE by taking parameter (vm, or vmThread, or either variant).
If it is frequently used (in inline allocation or wherever), the caller class can cache it.

Reply:
Ah I see, yes I will change this.